spec/services/attribute_asserter_spec.rb

```diff
@@ -101,6 +102,18 @@
     )
     CGI.unescape ialmax_authnrequest.split('SAMLRequest').last
   end
+  let(:raw_biometric_preferred_ial_authn_request) do
```

Comment:
do we need a `raw_biometric_requested_ial_authn_request` as well?

Reply:
Did you mean "required" as well as a test for it?

Reply:
yes, i do

Reply:
Adding test.